src/middlewared/middlewared/plugins/etc.py

```diff
@@ -213,12 +213,8 @@ def __init__(self, *args, **kwargs):
             'mako': MakoRenderer(self),
             'py': PyRenderer(self),
         }
-        self.args = tuple()
 
-    def get_args(self):
-        return self.args
-
-    async def generate(self, name, *args):
+    async def generate(self, name):
```

Comment:
etc plugin `loader` is also using the args. We would have to take care of that as well

Reply:
Thanks, missed that.

Reply:
No problem :)